libs/ui/src/pure/Footer/index.tsx

```diff
@@ -296,6 +289,7 @@ export const Footer = ({
   maxWidth,
   host,
 }: FooterProps) => {
+  const cowAnalytics = useCowAnalytics()
```

Comment:
Another pure component 🙈 

Maybe best move them to containers or something in case there's no easy solution?

Reply:
Yes, good point will do

Reply:
599906b707d5d0b8e9ca845b8c3069ccd3c8baa5